src/client/styles/components/_form-dropdown.scss

```diff
@@ -7,7 +7,7 @@
   padding: $baseline-grid-unit;
   width: 100%;
   line-height: 32px;
-  font-size: 30px;
+  font-size: 20px;
```

Comment:
You don't need this any more if not setting a larger font size. `@include core-font(20)` is setting the font to `20px`.